src/main/java/uk/gov/hmcts/divorce/systemupdate/event/SystemProgressCaseToAosOverdue.java

```diff
@@ -31,8 +31,8 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
         new PageBuilder(configBuilder
             .event(SYSTEM_PROGRESS_TO_AOS_OVERDUE)
             .forStateTransition(of(AwaitingAos, AosDrafted), AosOverdue)
-            .name("AoS Overdue")
-            .description("AoS Overdue")
+            .name("AOS Not Received within SLA")
```

Comment:
Can we go with `AoS not received within SLA`

Reply:
sure that looks correct. I had just copied from ticket. :)